scripts/weeklyupdate

```diff
@@ -2,58 +2,10 @@
 #set -x  # for debugging, prints each line before executing it
 
 # Jobs that need running weekly
+# Currently empty, and moved to dataupdate
```

Comment:
we should probably just remove this too and update the crontab to not run it.

Reply:
Yep.